docs/source/release_notes/release-notes.rst

```diff
@@ -34,39 +34,47 @@ Changelog
 develop
 =======
 
+.. <<<<------------------------------------------------------------------------------------------------------------->>>>
+
+=======
+v0.26.0
+=======
+
 .. list-table::
     :widths: 5 40
     :header-rows: 1
 
     *    - Type
          - Change
 
-    *    - |breaking|
```

Comment:
oh we ended up reverting this change. There's a few reasons we don't want to do it.